intelmq/bots/outputs/kafka/output.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+
+from json import loads
+from json import dumps
+from json import load
+from collections.abc import Mapping
+from intelmq.lib.utils import load_configuration
+from intelmq.lib.bot import Bot
+try:
+    from confluent_kafka import Producer
+except ImportError:
+    Producer = None
+try:
+    from confluent_kafka import avro
+except ImportError:
+    avro = None
+
+if avro is not None:
+    from confluent_kafka.avro import AvroProducer
```

Comment:
Is this for a different library version or why is it necessary? Please document the reason inline.

And this line is not in a try-except block.

Reply:
Did this again for better error handling as i was writing this. It can probably be removed as there's multiple python packages which fit within the kafka namespace. many of them have Producer class definitions, but no avro class definitions. confluent_kafka is a more specific namespace.

Reply:
> Did this again for better error handling as i was writing this.

But an ImportError is not catched here.